apps/src/javalab/javalabRedux.js

```diff
@@ -49,6 +58,11 @@ export const toggleDarkMode = () => ({
   type: TOGGLE_DARK_MODE
 });
 
+export const setIsEditingStartSources = isEditingStartSources => ({
```

Comment:
Would you mind adding this to the pageConstants redux store instead? Having this available in all apps will make it a lot easier to improve the start_sources experience across all apps. Additionally, the pageConstants store has protections against this value getting edited accidentally. 
PageConstants: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/redux/pageConstants.js
Where Java Lab adds values to the constants store: https://github.com/code-dot-org/code-dot-org/blob/b57f7785ca41999d7a63a48677801863761b3e62/apps/src/javalab/Javalab.js#L99